playbooks/aws/README.md

```diff
@@ -36,85 +56,120 @@ Before any provisioning may occur, AWS account credentials must be present in th
-
-Values specified in provisioning_vars.yml may instead be specified in your inventory group_vars
-under the appropriate groups.  Most variables can exist in the 'all' group.
+### provisioning_vars file
 
+The following are the minimum mandatory provisioning variables
 ```yaml
 ---
-# Minimum mandatory provisioning variables.  See provisioning_vars.yml.example.
-# for more information.
-openshift_deployment_type: # 'origin' or 'openshift-enterprise'
-openshift_release: # example: v3.7
-openshift_pkg_version: # example: -3.7.0
+# Minimum mandatory provisioning variables.
+# See provisioning_vars.yml.example for more information.
+openshift_aws_clusterid: # example: example
 openshift_aws_ssh_key_name: # example: myuser_key
 openshift_aws_base_ami: # example: ami-12345678
 # These are required when doing SSL on the ELBs
 openshift_aws_iam_cert_path: # example: '/path/to/wildcard.<clusterid>.example.com.crt'
 openshift_aws_iam_cert_key_path: # example: '/path/to/wildcard.<clusterid>.example.com.key'
+# Mandatory csv list of docker devices
+container_runtime_docker_storage_setup_device: '/dev/xvdb'
+# These are required when doing an openshift-enterprise installation
+rhsub_user: # example: 'myrhsub_user@example.com'
+rhsub_pass: # example: 'myrhsub_password'
+rhsub_pool: # example: 'abcd1234abcd1234abcd1234abcd1234'
 ```
 
+Values specified in provisioning_vars.yml may instead be specified in your inventory group_vars
+under the appropriate groups.  Most variables can exist in the 'all' group.
+
 If customization is required for the instances, scale groups, or any other configurable option please see the ['openshift_aws/defaults/main.yml'](../../roles/openshift_aws/defaults/main.yml) for variables and overrides. These overrides can be placed in the `provisioning_vars.yml`, `inventory`, or `group_vars`.
 
-In order to create the bootstrap-able AMI we need to create a basic openshift-ansible inventory.  This enables us to create the AMI using the openshift-ansible node roles.  This inventory should not include any hosts, but certain variables should be defined in the appropriate groups, just as deploying a cluster
-using the normal openshift-ansible method.  See provisioning-inventory.example.ini for an example.
+### inventory file
 
-There are more examples of cluster inventory settings [`here`](../../inventory/).
+In order to create the bootstrap-able AMI we need to create a basic openshift-ansible inventory.  This enables us to create the AMI using the openshift-ansible node roles.  Certain plays and tasks should execute on the local host as the local user.  It is highly recommended adding the folloing line to the inventory file.
```

Comment:
This would be better as "directions" rather than "suggestions".  Maybe changing the instances of `it is highly recommended...` with `Add the following lines to the inventory:`